notify.py

```diff
@@ -1,6 +1,10 @@
 import ctypes
 import time
-from toast import ToastNotifier
+try:
+    from toast import ToastNotifier
+except:

```

Comment:
This exception is too broad, please change it to a specific module related exception.